### PR TITLE
ICMSLST-466 Management commands to create importer/exporter

### PR DESCRIPTION
**Pull request by kevinetienne**
Usage:

```bash
$ make manage COMMAND="create_importer"
$ make manage COMMAND="create_exporter"
$ make manage COMMAND="create_exporter --first_name=John --last_name=McCarthy"
```

### CODE REVIEW

File: web/management/commands/create_exporter.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from web.domains.exporter.models import Exporter
+from web.domains.office.models import Office
+from web.domains.user.models import User
+
+
+class Command(BaseCommand):
+    help = """Create an exporter associated with an Export Company"""
+
+    def add_arguments(self, parser):
+        parser.add_argument("-first_name", nargs="?", default="Ada")
```
Comment (oskusalerma) [inline web/management/commands/create_exporter.py:14]: should be `--first-name` and `--last-name`

that's the convention for command-line args

File: web/management/commands/create_exporter.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        parser.add_argument("-last_name", nargs="?", default="Lovelace")
+        parser.add_argument("-title", nargs="?", default="Super")
```
Comment (oskusalerma) [inline web/management/commands/create_exporter.py:16]: if I saw "Super" i'd never think it's a title...what about `Ms`?

File: web/management/commands/create_importer.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+from web.domains.importer.models import Importer
+from web.domains.office.models import Office
+from web.domains.user.models import User
+
+
+class Command(BaseCommand):
+    help = """Create an importer associated with an Import Company"""
+
+    def add_arguments(self, parser):
+        parser.add_argument("-first_name", nargs="?", default="Guido")
```
Comment (oskusalerma) [inline web/management/commands/create_importer.py:14]: same as in the other one

**Review by oskusalerma — CHANGES_REQUESTED** [review]

File: web/management/commands/create_importer.py (hunk 3 above)
Comment (kevinetienne) [reply web/management/commands/create_importer.py]: Updated to `Mr` in https://github.com/uktrade/icms/pull/225/commits/e20fe6fa8ad0da0f9a853482abb21e3bb945b9c4

File: web/management/commands/create_exporter.py (hunk 1 above)
Comment (kevinetienne) [reply web/management/commands/create_exporter.py]: Updated for both commands in https://github.com/uktrade/icms/pull/225/commits/e20fe6fa8ad0da0f9a853482abb21e3bb945b9c4

File: web/management/commands/create_exporter.py (hunk 2 above)
Comment (kevinetienne) [reply web/management/commands/create_exporter.py]: Updated to `Ms` in https://github.com/uktrade/icms/pull/225/commits/e20fe6fa8ad0da0f9a853482abb21e3bb945b9c4

**Review by oskusalerma — APPROVED** [review]

**Pull request merged**